### PR TITLE
Add bookinfo traffic shifting tests

### PR DESCRIPTION
**Pull request by vadimeisenbergibm**
The tests test shifting traffic for a normal user gradually from v1 to v2, first 10% is shifted to v2, then 20%  are shifted to v2.

Added html for the traffic of normal user to v2.

### CODE REVIEW

**Comment by ldemailly** [comment]
why is there new html ? can you fill up the PR description ?

**Comment by vadimeisenbergibm** [comment]
@ldemailly I have added the description.

**Comment by vadimeisenbergibm** [comment]
@ldemailly Could you please approve the PR?

**Comment by ldemailly** [comment]
I don't understand why you need a brand new html ?

**Comment by vadimeisenbergibm** [comment]
@ldemailly You have several htmls in https://github.com/istio/istio/tree/master/tests/apps/bookinfo/output:

a different html per combination of user/version of the _reviews_ microservice. The htmls per version are different since you have different features in the html, for example stars with black or red color. The htmls per user are different since you have either _test_user_ logged in:
```html
<p class="navbar-text navbar-right">
            <i class="glyphicon glyphicon-user" aria-hidden="true"></i>
            <span style="padding-left: 5px;">test-user  ( <a href="logout">sign out</a> )</span>
</p>
```

or _normal user_ logged in:
```html
 <p class="navbar-text navbar-right">
            <i class="glyphicon glyphicon-user" aria-hidden="true"></i>
            <span style="padding-left: 5px;">normal-user  ( <a href="logout">sign out</a> )</span>
</p>
```

I add a new combination (normal user, v2) to test, so I have to add a new html.

**Comment by ldemailly** [comment]
There should be a way to templatize those instead of copy-pasting big files but I guess... future cleanup (or I can't wait we move bookinfo out of istio/istio)

File: tests/apps/bookinfo/output/productpage-normal-user-v2.html [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+        
+        <p class="navbar-text navbar-right">
+            <i class="glyphicon glyphicon-user" aria-hidden="true"></i>
+            <span style="padding-left: 5px;">normal-user  ( <a href="logout">sign out</a> )</span>
+        </p>
+        
+    </div>
+</nav>
+
+<!---
```
Comment (ldemailly) [inline tests/apps/bookinfo/output/productpage-normal-user-v2.html]: at least remove that block ?

**Comment by ldemailly** [comment]
/lgtm
under duress :-)

File: tests/apps/bookinfo/output/productpage-normal-user-v2.html (hunk 1 above)
Comment (vadimeisenbergibm) [reply tests/apps/bookinfo/output/productpage-normal-user-v2.html]: This is part of bookinfo implementation, the output html reflects the actual output of the application.

**Comment by vadimeisenbergibm** [comment]
@ldemailly 

> There should be a way to templatize those instead of copy-pasting big files but I guess... future cleanup (or I can't wait we move bookinfo out of istio/istio)

Let me fix it later, as part of bookinfo restructuring. Regarding moving bookinfo out of istio/istio, in general I am in favor of putting everything in a separate repo, and I did not like the merge of the Istio repositories into one. If there will be a decision to move bookinfo out of Istio, I would be glad to participate in its implementation.

**Pull request merged**